voter-dapp/src/ActiveRequests.js

```diff
@@ -185,7 +249,44 @@ function ActiveRequests() {
       return { statusString: "Cannot be revealed", currentVote: "", enabled: false };
     }
   });
-  const revealButtonEnabled = statusDetails.some(statusDetail => statusDetail.enabled);
+  const revealButtonShown = votePhase.toString() === VotePhasesEnum.REVEAL;
+  const revealButtonEnabled =
+    votePhase.toString() === VotePhasesEnum.REVEAL && statusDetails.some(statusDetail => statusDetail.enabled);
```

Comment:
nit: why not just use revealButtonShown instead of putting the same equality test in revealButtonEnabled's declaration?

Side not: I'm assuming it doesn't really matter if revealButtonEnabled is true when in the commit phase since it isn't shown, so we technically don't need to test the phase condition, right? I'm assuming that was added for readability? Or am I missing something?

Reply:
Done, can simply remove the first check.